src/evo/cbtx.cpp

```diff
@@ -230,7 +230,7 @@ auto CachedGetQcHashesQcIndexedHashes(const CBlockIndex* pindexPrev, const llmq:
 auto CalcHashCountFromQCHashes(const QcHashMap& qcHashes)
 {
     size_t hash_count{0};
-    for (const auto& [_, vec_hashes] : qcHashes) {
+    for (const auto& [__, vec_hashes] : qcHashes) {
```

Comment:
Better to keep `_` because:
>  P2169 proposes the underscore (_) as placeholder for unused variables in a backwards compatible manner. The advantage is that this would standardize common practice and works without changes on most existing code. 

https://www.reddit.com/r/cpp/comments/10dieeo/a_nice_placeholder_with_no_name_p2169/?rdt=48723

Reply:
> This causes a conflict between the normal usage of underscore as a
placeholder and Bitcoin's usage of it as a function name, which is
reported by the Dash-specific linter.

can make an exception for our linter specifically for this case

Reply:
P2169 means that it's being considered as a potentially official placeholder, thanks for bringing it to light! Resolved in latest push!